Apps/TimelineApp/core/sensor/src/main/java/uk/ac/cam/cares/jps/sensor/SensorService.java

```diff
@@ -272,18 +267,18 @@ public void run() {
      * @throws IOException If an I/O error occurs during the compression process.
      */
     public static byte[] compressData(String data) throws IOException {
```

Comment:
this has been defined in SensorUploadWorker, move to some util class as a static method